### ISSUE TITLE
Is cross platform support planned (Linux)

### ISSUE THREAD

**Issue by mweibel** [codeperfect95/issue-tracker#3]
### What is your question?

I was wondering if you plan to support other OS, namely Linux?

**Comment by invrainbow** [comment]
Yep, porting to Windows and Linux is a work in progress, and since there seems to be demand for it, it's now a fairly big priority.

I want to say it's hopefully about a month of work, optimistically speaking. We're not starting from scratch — we actually used to support Windows and and Linux was 95% of the way there, but then my PC died.
